README.md

```diff
@@ -659,6 +659,56 @@ registration time instead of `describe`. If this could cause problems, either
 implement a proper `describe`, or if that's not practical have `describe`
 return an empty list.
 
+### DropwizardExports Collector
+
+DropwizardExports collector is available to proxy metrics from Dropwizard.
+
+```java
+// Dropwizard MetricRegistry
+MetricRegistry metricRegistry = new MetricRegistry();
+new DropwizardExports(metricRegistry).register();
+```
+
+By default Dropwizard metrics are translated to Prometheus sample sanitizing their names, i.e. replacing unsupported chars with `_`, for example:
+```
+Dropwizard metric name:
+org.company.controller.save.status.400
+Prometheus metric:
+org_company_controller__save_status_400
+```
+
+It is also possible add custom labels and name to newly created `Sample`s by using a `CustomMappingSampleBuilder` with custom `MapperConfig`s:
+
+```java
+// Dropwizard MetricRegistry
+MetricRegistry metricRegistry = new MetricRegistry();
+MapperConfig config = new MapperConfig();
+// The match field in MapperConfig is a simplified glob expression that only allows * wildcard.
+config.setMatch("org.company.controller.*.status.*");
+// The new Sample's template name.
+config.setName("org.company.controller");
+Map<String, String> labels = new HashMap<String,String>();
+// ... more configs
+// Labels to be extracted from the metric. Key=label name. Value= label template
+labels.put("name", "${0}");
+labels.put("status", "${1}");
+config.setLabels(labels);
+
+SampleBuilder sampleBuilder = new CustomMappingSampleBuilder(Arrays.asList(config, ...));
+new DropwizardExports(metricRegistry, sampleBuilder).register();
+```
+
+When a new metric comes to the collector, `MapperConfig`s are scanned to find the first one that matches the incoming metric name. The name set in the configuration will
+be used and labels will be extracted. Using the `CustomMappingSampleBuilder` in the previous example leads to the following result:
+```
+Dropwizard metric name
+org.company.controller.save.status.400
+Prometheus metric
+org_company_controller {"name": "save", "status": "400"}
```

Comment:
This is not valid Prometheus text format

Reply:
Fixed